arangod/Aql/CollectNode.cpp

```diff
@@ -124,7 +123,6 @@ void CollectNode::toVelocyPackHelper(VPackBuilder& nodes, unsigned flags,
     }
   }
 
-  nodes.add("count", VPackValue(_count));
```

Comment:
Not sure yet if we should keep reporting this attribute for compatibility reasons, and simply always return "false" here until no clients (e.g. `db._explain(...)`) still look for it.
I suggest we keep it, but hard-code the return value to "false" here and leave a code comment about removing the attribute from the response in a future version.